bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -28,75 +28,76 @@ <h2 id="payouts-section">Payouts</h2>
     <tbody>
       <tr>
         <th scope="row">Remote Code Execution</th>
+        <td>$20041.19<sup>2</sup></td>
```

Comment:
I'm assuming there's some special meaning here, can we say why?

Reply:
See the '2' - thats an indication to look below...

Reply:
I think it's a bit overly cute, but if people really want it I'm okay with it.  :)

Reply:
Another option, since we're nearing the 15 years of Firefox, would be to set the RCE payouts to 15000 in celebration.

Reply:
I prefer the $15K proposal more.

Reply:
I do too.